change/@ni-nimble-components-a5fd15e6-373b-4e3c-b562-fc2e440df26c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add token to allow the table's height to be configured to the height of all table rows",
```

Comment:
@mollykreis gave another round of feedback but wasn't sure if PR still in work / intended to still be in draft.